proposals/description/tip_121a.ts

```diff
@@ -0,0 +1,169 @@
+import { ethers } from 'ethers';
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+const tip_121a: TemplatedProposalDescription = {
+  title: 'TIP-121a: PCV Consolidation, part 1',
+  commands: [
+    // 1. WETH -> DAI auction
+    {
+      target: 'ethToDaiLBPSwapper',
+      values: '0',
+      method: 'setReceivingAddress(address)',
+      arguments: (addresses) => [addresses.daiHoldingPCVDeposit],
+      description: 'Set the target of the WETH>DAI auction'
+    },
+    {
+      target: 'compoundDaiPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: (addresses) => [
+        addresses.ethToDaiLBPSwapper, // to
```

Comment:
Why are we withdrawing DAI to the WETH -> DAI swapper?

Reply:
Ah, okay it's to seed it - we later move the WETH from the holding deposit into this swapper

Reply:
correct